### PR TITLE
regenerated xtend code & cleanup imports

### PR DESCRIPTION
**Pull request by cdietrich**
regenerated xtend code & cleanup imports
Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

File: org.eclipse.xtext.ide/xtend-gen/org/eclipse/xtext/ide/server/TopologicalSorter.java [hunk 1]
```diff
@@ -55,8 +55,8 @@ public Entry(final ProjectDescription description) {
     List<ProjectDescription> _xblockexpression = null;
     {
       this.cyclicAcceptor = new IAcceptor<ProjectDescription>() {
-          public void accept(ProjectDescription t) {
-            cyclicAcceptor.apply(t);
+          public void accept(ProjectDescription arg0) {
+            cyclicAcceptor.apply(arg0);
```
Comment (miklossy) [inline org.eclipse.xtext.ide/xtend-gen/org/eclipse/xtext/ide/server/TopologicalSorter.java]: Why this change?

Comment (cdietrich) [reply org.eclipse.xtext.ide/xtend-gen/org/eclipse/xtext/ide/server/TopologicalSorter.java]: not intended

Comment (cdietrich) [reply org.eclipse.xtext.ide/xtend-gen/org/eclipse/xtext/ide/server/TopologicalSorter.java]: (this is a known xtend standalone bug)
